packages/nimbus/src/components/link/link.mdx

```diff
@@ -11,67 +11,215 @@ menu:
 tags:
   - component
 figmaLink: >-
-  https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?node-id=384-5726&m=dev
+  https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1696-49827&m
 ---
 
 # Link
 
-Allows a user to navigate to a different page or resource.
+---
+
+A link is a clickable element that navigates users to another page, section of the page, or resource, and is visually distinguished by styling.
+
+## Overview
+
+The link is flexible to many uses and matching text styles for common placements. Most commonly used in dialog boxes, alerts, and within body copy.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[TECH RELATED LINK](https://react-spectrum.adobe.com/react-spectrum/Link.html#link)
+[TECH RELATED LINK](https://www.w3.org/WAI/ARIA/apg/patterns/link/)
```

Comment:
Remove for now? I like the idea of linking to w3 and the React Aria documentation. Care amount is so low, we can adjust in cleanup.